Ecosystem Partners/Marketplace Guides/gettting-started-with-biztalk-distributed.md

```diff
@@ -0,0 +1,229 @@
+3. **Customize the Blueprint**
+	1. Set the AD/DNS Server properties for the AD/DNS server.
+
+        'Execute on Server' - set to the provisioned AD/DNS server name
+
+        'Execute on Server' - set to the provisioned AD/DNS server name
+
+        'Domain Name'  - set to your domain name
+
+        'Net BIOS Name' - set to your domain Net BIOS name
+
+        'Safe Mode Admin Password' - set to your domain administrator password
+
+        'Confirm Safe Mode Admin Password' - set to your domain administrator password
+
+		<div class="row">
+		  <a><img src="../../images/AD_DNS_Server.png" width=300></a>
+		</div>
+	2. Set the SQL/ENTSSO Server properties for the SQL Server and Enterprise Single Signon server.
+
+        2.1 Set the SQL Server properties.
+
+        'Execute on Server' - set to the provisioned SQL/SSO server name
+
+        'SA Password' - SQL System Administrators password
+
+        'Confirm SA Password' - Confirm SQL System Administrators password
+
+        'SQL Version' - '2012 Standard' or 'Enterprise'
+
+        'Database Engine Services' - check this feature
+
+        'Management Tools' - check this feature
+
+        'Connectivity Components' - check this feature
+
+		<div class="row">
+		  <a><img src="../../images/SQL_Server.png" width=300></a>
+		</div>
+        2.2 Set the ENTSSO Server properties.
+
+        'Execute on Server' - set to the provisioned SQL/SSO server name
+
+        'Company Name' - set to your company name
+
+        'User Name' - set to your name
+
+		<div class="row">
+		  <a><img src="../../images/ENSSSO_Server.png" width=300></a>
+		</div>
+        2.3 Set the Join SQL/ENTSSO server to the domain.
+
+        'Execute on Server' - set to the provisioned SQL/SSO server name
+
+        'Domain Name' - Set to the new Domain Name
+
+        'DNS Server Name' - set to the AD/DNS server name
+
+        'DNS Primary IP Address' - set to the IP addres of the AD/DNS server
+
+        'Domain Admin Password' - set to your domain administrator password
+
+        'Confirm Domain Admin Password' - set to your domain administrator password
+
+		<div class="row">
+		  <a><img src="../../images/Join_SQLSSO_Server.png" width=300></a>
+		</div>
+	5. Set the BTS Runtime server properties.
+
+        'Execute on Server' - set to the provisioned BizTalk Runtime server name
+
+        'Execute on Server' - set to the provisioned BizTalk Runtime server name
+
+        'Company Name' - set to your company name
+
+        'User Name' - set to your name
+
+        'Execute on Server' - set to the provisioned BizTalk Runtime server name
+
+        'Domain Name' - Set to the new Domain Name
+
+        'DNS Server Name' - set to the AD/DNS server name
+
+        'DNS Primary IP Address' - set to the IP addres of the AD/DNS server
+
+        'Domain Admin Password' - set to your domain administrator password
+
+        'Confirm Domain Admin Password' - set to your domain administrator password
+
+		<div class="row">

```

Comment:
it would be much better if all your image names matched the kb they linked to.  

i.e. gettting-started-with-biztalk-distributed-01.png, gettting-started-with-biztalk-distributed-02.png etc.  just having random names don't really tie things together if someone is trying to figure out what KB's are tied to what images.  I'll defer to #digital if this is a must prior to merging but its what most of our team tries to do as a best practice.